### DISCUSSION TITLE
Seismic module for FAST v7

### DISCUSSION THREAD

**Discussion by tolgasarikaya** [OpenFAST/openfast#1166] [General]
Hi everyone,
I need seismic archive Seismic_v2.00.00.exe for my simulations, however I could not able to find it. If you have it could you please send me? My e-mail adress : tolga.sari.kaya@hotmail.com
Thanks a lot.

**Comment by jjonkman** [comment]
Dear @tolgasarikaya,

I've sent you the Seismic archive via WeTransfer.

Best regards,

**Reply by tolgasarikaya** [reply]
Thank you very much. Have a good day.

**Comment by zhao12545** [comment]
Hi everyone，
First,I need seismic archive Seismic_v2.00.00.exe for my simulations,If you have it could you please send me?
Then,I want to obtain the FASTv7 case about Seismic. The FASTv7 I downloaded from NREL does not contain the example of Seismic file（may be i had not found it）Please help me.([zhaohailong0222@163.com],Thanks !
Best regards,

**Reply by jjonkman** [reply]
Dear @zhao12545,

I've sent you the Seismic archive of FAST v7 via WeTransfer.  This archive contains example models.

Best regards,

**Reply by zhao12545** [reply]
Thank you~~~

**Comment by zhao12545** [comment]
Dear Jackson ,
Thank you! By the way, Merry Christmas!

Best wishes ,

---- Replied Message ----
| From | ***@***.***> |
| Date | 12/28/2023 23:23 |
| To | OpenFAST/openfast ***@***.***> |
| Cc | zhao12545 ***@***.***>,
Mention ***@***.***> |
| Subject | Re: [OpenFAST/openfast] Seismic module for FAST v7 (Discussion #1166) |

Dear @zhao12545,

I've sent you the Seismic archive of FAST v7 via WeTransfer. This archive contains example models.

Best regards,

**Comment by Fouriergithub** [comment]
Hello everyone，
To begin with, I need seismic archive Seismic_v2.00.00.exe , Could u please send it to me? Very much thanks.
Second,I also nedd the FASTv7 case about Seismic. Honestly speaking i also didnt find the case as Zhao mentioned before. Please help me
My email: qsr20121518@163.com
Thanks !
Best regards.

**Reply by jjonkman** [reply]
Dear @Fouriergithub ,

I've sent you the Seismic archive of FAST v7 via WeTransfer. This archive contains example models.

Best regards,

**Reply by Fouriergithub** [reply]
Thank you soooo much

**Comment by zxh804159854** [comment]
Hi everyone,
I am trying to simulate a turbine subjected in seismic motion, could you share me the Seismic_v2.00.00.exe and the FASTv7 case about Seismic. My e-mail adress : zxh804159854@163.com
Thanks a lot.

**Reply by jjonkman** [reply]
Dear @zxh804159854,

I've sent you the Seismic archive of FAST v7 via WeTransfer. This archive contains example models.

Best regards,

**Reply by zxh804159854** [reply]
Dear Jackson ,
Thank you! 

Best wishes ,

**Comment by ZJU-Wangyi** [comment]
Hello everyone，
To begin with, I need seismic archive Seismic_v2.00.00.exe , Could u please send it to me? Very much thanks.
Second,I also nedd the FASTv7 case about Seismic. Honestly speaking i also didnt find the case as Zhao mentioned before. Please help me
My email:zjuwy1@163.com
Thanks !
Best regards.

**Reply by jjonkman** [reply]
Dear @ZJU-Wangyi ,

I've sent you the Seismic archive of FAST v7 via WeTransfer. This archive contains example models.

Best regards,
